### DISCUSSION TITLE
come capire se posso installare valvola smart ai termo attuali

### DISCUSSION THREAD

**Discussion by SkunkGuru** [indomus/forum#2770] [GENERALE - Discussioni generiche]
Ciao a tutti,
stavo pensando di installare delle valvole smart, ma prima di capire quale modello acquistare, volevo capire se effettivamente posso installarle nella situazione in cui mi trovo. Impianto autonomo, attualmente ho soltanto una rotella di apertura/chiusura manuale, nessuna ghiera numerata o altro. Ovviamente l'impianto è precedente per cui non ho riferimenti di ditte, modelli o altro.

Controllando 2 termosifoni mi pare che siano pure fatte diverse. La marca è RBM per entrambe.
Nelle prime 3 foto il primo termo, è presente una 'E' sul davanti e una specie di vite all'interno del tappo
Le seconde 2 foto l'altro termo, sul davanti c'è una 'I' e il meccanismo interno sembra diverso, pur con tappo uguale

Sapete darmi qualche info o qualche dritta?

termo 1
![photo_2023-12-13_21-40-28](https://github.com/indomus/forum/assets/15073199/910435b0-54d8-4ba7-a157-9ddbf7423030)
![photo_2023-12-13_21-40-47](https://github.com/indomus/forum/assets/15073199/d32b7a6e-0be9-47b8-862a-1b908abd60b0)
![photo_2023-12-13_21-40-42](https://github.com/indomus/forum/assets/15073199/17c8ea27-8e16-4ff3-99ca-b27e905a9ac9)

termo 2
![photo_2023-12-13_21-43-29](https://github.com/indomus/forum/assets/15073199/ed9c37ec-3817-4898-8418-1eda2f42b120)
![photo_2023-12-13_21-43-22](https://github.com/indomus/forum/assets/15073199/2dbde3fa-418a-4bea-b157-e7af55d10e6d)

**Comment by Scialla** [comment]
Prova a vedere qui:
https://support.tado.com/it/articles/3405446-qual-e-la-differenza-tra-valvole-termostatizzabili-e-valvole-manuali-con-quali-tipologie-di-valvole-e-compatibile-tado

dovrebbero essere M30x1,5 verifica, se così fosse sono compatibili senza adattatori

**Comment by 50494554524F** [comment]
se smontata la valvola sotto c'è un pistoncino puoi montare quelle smart, se c'è un'altra vite no... sembrerebbe essere una vite

**Comment by dante70** [comment]
a me sembra il contrario.....mi sembra che la vite plastica agisca sul pistoncino.Assicurati con foto e almeno la ghiera che hai postato da un fornitore di materiale idraulico che effettivamente accettino la testa termostatica.Se cosi fosse ovviamente valuti l'acquisto delle smart

**Comment by KC-inDomus** [comment]
> Sapete darmi qualche info o qualche dritta?

Devi chiamare un idraulico e farti cambiare la valvola.

**Comment by SkunkGuru** [comment]
ho smontato anche l'altra parte bianca, ed effettivamente sotto c'è il pistoncino, a questo punto direi che la vita bianca serviva per spingere il pistoncino, e quindi sono compatibili con le soluzioni smart
grazie a tutti 

![photo_2023-12-16_15-02-51](https://github.com/indomus/forum/assets/15073199/a88e7665-cab3-4ec3-9c48-25d7cea74fb9)
